core/scss/core/fonts/_fonts.scss

```diff
@@ -19,6 +19,9 @@
 // Script font: Kalam.
 @import url('https://fonts.googleapis.com/css?family=Kalam:300,400,700');
 
+// Sandscrit font: Noto Sans
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,400italic,700,700italic');
```

Comment:
```suggestion
@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i&subset=devanagari');
```

Reply:
It seems like by default it only loads the latin subset so if we need sanskrit support we might need to add subset=devanagari at the end.